src/main/scripts/run_e2e_test_module.py

```diff
@@ -39,10 +39,9 @@ def run_shell_command(cmd):
     os.system("export JAVA_OPTS=-Xmx24G")
     run_shell_command(f"sandbox/{sandbox_dir}/bin/cdap sandbox start")
    
-
+    
     # Build the plugin
     os.chdir("plugin")
-
     module_to_build : str = sys.argv[1]
```

Comment:
Can we use optional named args instead of positional args?

Reply:
I will implement it in separate PR